api/src/pcapi/core/finance/repository.py

```diff
@@ -452,13 +457,188 @@ def _get_reimbursement_details_from_invoices_base_query(invoice_ids: list[int])
                 models.Pricing.status == models.PricingStatus.INVOICED,
             ),
         )
-        .join(models.Cashflow.bankInformation)
+        .join(models.Cashflow.bankAccount)
         .join(models.Cashflow.batch)
         .outerjoin(models.Pricing.customRule)
     )
 
 
 def _get_collective_booking_reimbursement_data(query: BaseQuery) -> list:
```

Comment:
Détail parce qu'il faut bien trouver des trucs à dire :slightly_smiling_face: 
```suggestion
def _get_collective_booking_reimbursement_data(query: BaseQuery) -> list[tuple]:
```